src/init/init.ts

```diff
@@ -169,10 +157,29 @@ function getDisplayName(generatorName: string) {
 function createYeomanEnvironment(): Promise<any> {
   return new Promise((resolve, reject) => {
     const env = new YeomanEnvironment();
+    // Element Template
     env.registerStub(ElementGenerator, 'polymer-init-element:app');
+    // Application Template
     env.registerStub(ApplicationGenerator, 'polymer-init-application:app');
-    env.registerStub(shopGenerator, 'polymer-init-shop:app');
-    env.registerStub(pskGenerator, 'polymer-init-starter-kit:app');
+    // Shop Template
+    // TODO: Add Shop ^2.0.0 generator once Shop for Polymer 2.0 template is released
+    env.registerStub(
+        createGithubGenerator({
+          owner: 'Polymer',
+          repo: 'shop',
+          semverRange: '^1.0.0',
+        }),
+        'polymer-init-shop:app');
+    // Polymer Starter Kit Template
+    // TODO: Add Shop ^3.0.0 generator once PSK for Polymer 2.0 template is released
```

Comment:
@robdodson Just want to confirm that this in line with what you were thinking for PSK release versioning:

- The current PSK v2.x branch keep Polymer 1.0 with Hybrid elements
- PSK v3.0 and onward get Polymer 2.0

Our plan is to keep this in for the first few versions after Polymer 2.0 release, and then phase it out once appropriate.